### PR TITLE
Create node-when-unrelated.yml

### PR DESCRIPTION
**Pull request by skjnldsv**
Not needed for 2.2.0

### CODE REVIEW

**Review by susnux — APPROVED** [review]
:+1:

File: .github/workflows/node-when-unrelated.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# Use node together with node-when-unrelated to make eslint a required check for GitHub actions
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+
+name: Node
+
+on:
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/**'
+      - 'src/**'
```
Comment (susnux) [inline .github/workflows/node-when-unrelated.yml:15]: Should it not be `lib/**`?

Comment (skjnldsv) [reply .github/workflows/node-when-unrelated.yml:15]: It's from the template :shrug:

Comment (skjnldsv) [reply .github/workflows/node-when-unrelated.yml:15]: I guess it doesn't fully apply to some npm packages :shrug:

Comment (susnux) [reply .github/workflows/node-when-unrelated.yml:15]: So written for apps not libs, but I think it should work by the last few rules so

**Review by susnux — APPROVED** [review]

**Pull request merged**